custom_model_runner/CHANGELOG.md

```diff
@@ -4,6 +4,10 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+#### [Current]
```

Comment:
Do we want to keep version as "Current" or do we want to bump the version in:
https://github.com/datarobot/datarobot-user-models/blob/c4f07b60e875af3da50e25d18b0d3f2a864d4c48/custom_model_runner/datarobot_drum/drum/description.py#L7

Reply:
I have two PRs open right now with changes, I will bump the version after both of those are in as its own PR.

Reply:
Sounds good thanks for the clarification 👍